ui/src/app/components/runtime-configuration-panel/offer-delete-disk-with-update.tsx

```diff
@@ -86,6 +86,7 @@ export const OfferDeleteDiskWithUpdate = ({
               style={{ display: 'inline-block', marginRight: '0.75rem' }}
             >
               <RadioButton
+                aria-label={'delete-pd'}
```

Comment:
This is not an appropriate ARIA label.  These should be formatted to be readable by users, not machines.  So a better choice here would be "Delete Persistent Disk"

Reply:
Done